lib/tests/function.rs

```diff
@@ -147,6 +172,46 @@ async fn function_array_append() -> Result<(), Error> {
 	Ok(())
 }
 
+// Skipping array_boolean_and since it's similar enough to 'or'.
```

Comment:
Could we add the tests for `array::boolean_and` even though it's similar to `array::boolean_or`

Reply:
sure! added in [c3c16ea](https://github.com/surrealdb/surrealdb/pull/2156/commits/c3c16ea3e5b90675a056d0f9339da8d2eeda3b93)